Makefile

```diff
@@ -113,6 +113,9 @@ clean:
 	rm -f kube-state-metrics
 	git clean -Xfd .
 
+lint-fix:
+       find . -name "*.go" ! -path "./vendor/*" | xargs goimports -local "k8s.io/kube-state-metrics/v2" -w
```

Comment:
Can we run `golangci-lint -fix` here? We can also add `k8s.io/kube-state-metrics/v2` to the `golangci.yml`: https://golangci-lint.run/usage/linters/#goimports

Reply:
Updated.

Reply:
I found my terminal was stuck after running this cmd.